lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -409,3 +415,26 @@ def get_all_treatment_history_data(connection, mrn):
     treatment_history = treatment_history.reset_index(drop=True)
 
     return treatment_history
+
+
+def get_structure_aliases():
+    file_path = pathlib.Path(__file__).parent.joinpath("ALIASES.json")
+    return pd.read_json(file_path)
+
+
+def add_new_structure_alias(dvh_calcs, alias_df):
+    file_path = pathlib.Path(__file__).parent.joinpath("ALIASES.json")
```

Comment:
Could you explain a bit what you mean by this? Define the variable once outside of the functions, and then call it as an argument in each?

Reply:
You could go for either:

```python
ALIAS_PATH = pathlib.Path(__file__).parent.joinpath("ALIASES.json")
```

or

```python
def get_alias_path():
    return pathlib.Path(__file__).parent.joinpath("ALIASES.json")
```

Downside of the top one is that it will run that logic on import, meaning if it takes some time to run then it slows down all of PyMedPhys when it is imported. Downside of the second one, it's a little bit more verbose to write.